torchvision/models/mobilenet.py

```diff
@@ -70,7 +70,12 @@ def forward(self, x):
 
 
 class MobileNetV2(nn.Module):
-    def __init__(self, num_classes=1000, width_mult=1.0, inverted_residual_setting=None, round_nearest=8):
+    def __init__(self,
+                 num_classes=1000,
+                 width_mult=1.0,
+                 inverted_residual_setting=None,
+                 round_nearest=8,
+                 InvertedResidual=InvertedResidual):
```

Comment:
can you make it `block=InvertedResidual` instead of `InvertedResidual=InvertedResidual`? In the current implementation, one obfuscates the other.